### DISCUSSION TITLE
Using Hono as the backend functions for your Cloudflare Pages SPA?

### DISCUSSION THREAD

**Discussion by carcinocron** [honojs/hono#3538] [Q&A]
I couldn't get anything from https://github.com/honojs/hono/issues/1247 where both the SPA and the hono both work at the same time. My SPA is a vite+solidjs but I don't think the front-end framework matters that much because it's just static files.

**Comment by maou-shonen** [comment]
You can add the [@hono/vite-build/cloudflare-workers](https://github.com/honojs/vite-plugins/tree/main/packages/build) adapter in Vite,
and build it independently into a separate output, such as dist-api.
Then, upload dist-api to cloudflare workers.
